src/channel_server/cmsg_header.hpp

```diff
@@ -80,13 +82,21 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #define CMSG_SEND_NOTE 0x81
 #define CMSG_MYSTIC_DOOR_ENTRY 0x83
 #define CMSG_KEYMAP 0x85
+#define CMSG_STALK_BEGIN 0x8c // Click on yourself in the minimap, and you'll send this packet.
+#define CMSG_ALLIANCE 0x8d
+#define CMSG_ALLIANCE_DENIED 0x8e
```

Comment:
Are all of these necessary? It seems like some of these are related to your other PR?